internal/snips/file.go

```diff
@@ -58,3 +61,33 @@ func (f *File) Visibility() string {
 
 	return "public"
 }
+
+var decoder, _ = zstd.NewReader(nil)
+
+func (f *File) GetContent() ([]byte, error) {
+	// check if it has been compressed
+	if !f.isCompressed() {
+		return f.content, nil
+	}
+
+	defer decoder.Close()
+	decodedBytes, err := decoder.DecodeAll(f.content, nil)
+
+	return decodedBytes, err
+}
+
+var encoder, _ = zstd.NewWriter(nil)
+
+func (f *File) SetContent(in []byte) {
+	f.content = encoder.EncodeAll(in, nil)
+}
```

Comment:
```suggestion
func (f *File) SetContent(in []byte, compress bool) {
	if compress {
		f.content = encoder.EncodeAll(in, nil)
	} else {
		f.content = in
	}
}
```

Adding a `compress` boolean to this method would be nice so that uses can "opt in" to using compression via a configuration option.

Reply:
Were you thinking of "opting in" via command line flags or environment variables? I was thinking environment variables  since I can't see why the users would have that option unless I'm missing something...